### PR TITLE
Add draft-collections

### PR DESCRIPTION
**Pull request by nsabri1**
While testing user journey for Collections-publisher` step by step`, discovered draft-collections was needed.

Error spotted on draft-router:
`draft-router-76b95c85f9-hq7vr app 2022-12-14T13:41:34.837072651Z {"@timestamp":"2022-12-14T13:41:34.836829873Z","@fields":{"error":"dial tcp 10.1.6.16:80: i/o timeout","request":"GET /nadeem-test?token=***HTTP/1.0","request_method":"GET","status":504,"upstream_addr":"draft-collections","varnish_id":""}}`

https://trello.com/c/259cJ1Ho/1054-journey-test-publish-a-step-by-step

### CODE REVIEW

File: charts/app-config/values-integration.yaml [hunk 1]
```diff
@@ -424,6 +424,9 @@ govukApplications:
             key: SECRET_KEY_BASE
       - name: REDIS_URL
         value: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+ - name: draft-collections
+   helmValues:
+     <<: *collections
```
Comment (sengi) [inline charts/app-config/values-integration.yaml:429]: Are you sure there isn't anything that needs setting differently for the draft version? (there must be, surely?)

Comment (nsabri1) [reply charts/app-config/values-integration.yaml:429]: looking at blue-frontend and blue-draft-frontend EC2 - i dont really see a difference other then maybe no need to use Memcache

Comment (sengi) [reply charts/app-config/values-integration.yaml:429]:
```diff
$ diff <(sshi frontend:1 -- govuk_setenv collections env | sort) <(sshi draft_frontend:1 -- govuk_setenv collections env | sort)
29d28
< MEMCACHE_SERVERS=frontend-memcached:11211
31a31,37
> PLEK_HOSTNAME_PREFIX=draft-
> PLEK_SERVICE_ACCOUNT_API_URI=https://account-api.integration.publishing.service.gov.uk
> PLEK_SERVICE_IMMINENCE_URI=https://imminence.integration.publishing.service.gov.uk
> PLEK_SERVICE_LICENSIFY_URI=https://licensify.integration.govuk-internal.digital
> PLEK_SERVICE_LOCAL_LINKS_MANAGER_URI=https://local-links-manager.integration.publishing.service.gov.uk
> PLEK_SERVICE_SEARCH_API_URI=https://search-api.integration.govuk-internal.digital
> PLEK_SERVICE_SEARCH_URI=https://search.integration.govuk-internal.digital
```

Comment (sengi) [reply charts/app-config/values-integration.yaml:429]: Memcache matters as much as Plek stuff btw — if we were to accidentally hook up draft-collections to memcache, we could easily expose pre-publication content on the public website.

Comment (sengi) [reply charts/app-config/values-integration.yaml]: You can copy from another draft app for the Plek stuff — you probably only need to set `PLEK_HOSTNAME_PREFIX` and the global settings from env-configmap should take care of the overrides for you.

Comment (nsabri1) [reply charts/app-config/values-integration.yaml]: Great - i just compared env vars rather then the Govuk_setenv - thanks updated

File: charts/app-config/values-integration.yaml [hunk 2]
```diff
@@ -424,6 +424,33 @@ govukApplications:
...
+      - name: PLEK_SERVICE_IMMINENCE_URI
+        value: http://imminence
+      - name: PLEK_SERVICE_LICENSIFY_URI
+        value: https://licensify.integration.govuk-internal.digital
+      - name: PLEK_SERVICE_LOCAL_LINKS_MANAGER_URI
+        value: http://local-links-manager
+      - name: PLEK_SERVICE_SEARCH_API_URI
+        value: http://search-api
+      - name: PLEK_SERVICE_SEARCH_URI
+        value: http://search
```
[the remark this answers is not in the document]
Comment (nsabri1) [reply charts/app-config/values-integration.yaml]: Oh yes - forgot to check default config map!!

[the remark this answers is not in the document]
Comment (nsabri1) [reply charts/app-config/values-integration.yaml]: well then just needs the PLEK_HOSTNAME_PREFIX set which i have left in

File: charts/app-config/values-integration.yaml [hunk 3]
```diff
@@ -424,6 +424,23 @@ govukApplications:
...
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: EMAIL_ALERT_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-collections-email-alert-api
+            key: bearer_token
+      - name: PLEK_HOSTNAME_PREFIX
+        value: draft-
+      - name: PLEK_SERVICE_LICENSIFY_URI
```
Comment (sengi) [inline charts/app-config/values-integration.yaml:442]: I don't think this one should be necessary here either

**Review by sengi — APPROVED** [review]

**Pull request merged**